src/civic/index.js

```diff
@@ -410,15 +416,21 @@ const processEvidenceRecord = async (opt) => {
         evidence: [rid(publication)],
         evidenceLevel: [rid(level)],
         relevance: rid(relevance),
-        reviewStatus: 'not required',
+        reviewStatus: (rawRecord.status === 'accepted'
+            ? 'not required'
+            : 'pending'
```

Comment:
just curious why this change is needed when we are only requesting 'accepted' evidence, other than from trusted curators

Reply:
This is not changing due to the added option and more just because it was previously misleading. Its more accurate to port the submitted not accepted entries in as pending